src/flavors/list.rs

```diff
@@ -1,12 +1,14 @@
 //! Unbounded channel implemented as a linked list.
 
+use std::alloc;
```

Comment:
Do we have to increment the minimum required rust version by using `std::alloc`? (I think a minimal subset has been stabilized in 1.28).

Reply:
Indeed - we'll have to bump it to 1.28.

Reply:
Yes I missed that in the `.travis.yml`. Fix coming soon!